test/rest_api/test_hinted_handoff.py

```diff
@@ -0,0 +1,21 @@
+import requests
+import urllib.parse
+import base64
+
+def test_hinted_handoff_sync_point(rest_api):
+    resp = rest_api.send('POST', "hinted_handoff/sync_point")
+    sync_point = resp.json()
+    sync_point_b = base64.b64decode(sync_point.encode('ascii'))
+
+    resp = rest_api.send('GET', "hinted_handoff/sync_point", { "id": urllib.parse.quote(sync_point) })
+    assert resp.ok
+
+    for i in range(1, len(sync_point_b)):
```

Comment:
Please add a comment which explains why we are skipping the first byte.

Reply:
The comment highlights that we do skip the first byte, but it doesn't answer the question why we do it.

Reply:
I've added an explanation.